### ISSUE TITLE
Create artwork for Bett Show handout leaflet

### ISSUE THREAD

**Issue by j4mesholland** [ocadotechnology/codeforlife-deploy-appengine#140]
## Description
We would like to have a handout leaflet that gives a summary for teachers of the benefits of using Code for Life.

## Acceptance criteria
- Size 1/3 A4 bifold printed both sides in full colour
- Must include the following content:
  - Curriculum links
  - Mention that we are free!
  - Assessment 
  - Teacher Testimonials 
  - Info about Ocado technology

**Comment by j4mesholland** [comment]
Please review the design and content:
[https://app.zenhub.com/files/49142916/7595d6f3-fd03-4c09-bb72-93f04ded3c81/download](https://app.zenhub.com/files/49142916/7595d6f3-fd03-4c09-bb72-93f04ded3c81/download)

**Comment by faucomte97** [comment]
Looks great @j4mesholland !
Just a couple of grammar mistakes to fix:
- at the bottom of the About Us page: we're putting the *world's* retailers together
- in the last page, the world *assessment* is missing an s twice
- could we maybe rephrase the mission statement by replacing the second 'that' with *'to'*? I'm just wondering because the two 'that' makes it a bit repetitive.
- about how CfL is aligned to the English curriculum, the asterisk mentions we're working on aligning to the rest of the UK. I thought we were also aligned to the Welsh curriculum? Just wanted to make sure :)

**Comment by CelineBoudier** [comment]
Nice!

A couple of remarks:
- I'm afraid adding a * after "free" will make people think there's a caveat. Maybe the explanation that it's really really free shouldn't be presented with an asterisk?
- about the curriculum, the wording is a bit awkward? Maybe matching the the rest of the UK curriculumS or something like this

I really like the layout
